README.md

```diff
@@ -93,6 +93,8 @@ CI for PR job in this repository will use `build.sh --pr`.
 If you want to publish docker images use `build.sh --push`
 
 ### How to build own che-theia image with Docker
+Make sure that you have added name of your branch into che-theia-init-sources.yml (`checkoutTo` section).
```

Comment:
Not a big deal but a best practice in markdown is to [surround headers with blank lines]( https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines)

Reply:
done